src/Augurk/templates/configuration.html

```diff
@@ -112,6 +112,46 @@ <h2>Customization &amp; Configuration</h2>
         </div>
     </div>
 
+    <div class="panel panel-default">
+        <div class="panel-heading">Import &amp; Export</div>
+        <div class="panel-body">
+            <div class="row" style="margin-top: 10px;">
+                <div class="col-md-6">
+                    <div class="panel panel-default">
+                        <div class="panel-heading">Import</div>
+                        <div class="panel-body">
+                            <p>
+                                Choose a file to import and then click the import button to restore the data from the
+                                selected file into this instance of Augurk.
+                            </p>
+                            <p>
+                                <strong>Note:</strong> This will <strong>overwrite</strong> any existing <strong>products,
```

Comment:
This doesn't feel complete. It only replaces them if matched right? Is that clear from this text?